src/main/java/com/hms_networks/americas/sc/extensions/historicaldata/HistoricalDataManager.java

```diff
@@ -236,6 +260,32 @@ public static Map readHistoricalFifo(
     return parseEBDHistoricalLogExportResponse(exporter, timeSpan);
   }
 
+  /**
+   * Reads historical log between <code>startTime</code> and <code>endTime</code>. Returns a map of
+   * rounded timestamps to lists of data points. <br>
+   * (Parameterized map type: Map&lt;Date, List&lt;DataPoint&gt;&gt;)
+   *
+   * <p>In version 3.0.0 and later, this method returns data points with an ISO 8601 formatted
+   * timestamp. Previous versions used a timestamp integer with the number of seconds since epoch,
+   * thus callers of this method must adapt as necessary.
+   *
+   * @param requestEBD historical data request EBD string
+   * @param timeSpan time span to round data point time stamps to
+   * @return data points from response
+   * @throws IOException if export block descriptor fails
+   * @throws JSONException if unable to parse int to string enumeration file
+   * @throws EbdTimeoutException for EBD timeout
+   * @throws IllegalArgumentException if time unit is null, unknown, or not supported
+   * @throws Exception if unable to parse data point timestamp to date
+   * @since 3.0.0
+   */
+  public static Map readHistoricalFifo(String requestEBD, SCTimeSpan timeSpan) throws Exception {
```

Comment:
The name `requestEBD` violates Google Code Format. It should be `requestEbd`.

Reply:
changed